src/simulation/hils/hils_port_manager.hpp

```diff
@@ -81,28 +81,28 @@ class HilsPortManager {
    * @fn I2cTargetReadRegister
    * @brief Read I2C register in S2E
    * @param [in] port_id: COM port ID
-   * @param [in] reg_addr: Register address to read
+   * @param [in] reg_address: Register address to read
    * @param [out] data: Data buffer to store the read data
-   * @param [in] len: Read data length
+   * @param [in] length: Read data length
    */
-  virtual int I2cTargetReadRegister(unsigned int port_id, const unsigned char reg_addr, unsigned char* data, const unsigned char len);
+  virtual int I2cTargetReadRegister(unsigned int port_id, const unsigned char reg_address, unsigned char* data, const unsigned char length);
```

Comment:
```suggestion
  virtual int I2cTargetReadRegister(unsigned int port_id, const unsigned char register_address, unsigned char* data, const unsigned char length);
```

[NITS] addressを直すなら，regも直したいですね．